package.json

```diff
@@ -9,21 +9,26 @@
     "apk": "cd android && ./gradlew assembleRelease",
     "rename": "node ./bin/rename.js",
     "start": "react-native start",
-    "test": "jest"
+    "test": "jest",
+    "lint": "eslint .",
+    "postinstall": "npx jetify"
   },
   "dependencies": {
-    "react": "16.8.3",
-    "react-native": "0.59.9",
-    "react-native-firebase": "^5.5.4"
+    "jetifier": "^1.6.4",
+    "react": "16.8.6",
+    "react-native": "0.60.5",
+    "react-native-firebase": "^5.5.6"
   },
   "devDependencies": {
-    "@babel/core": "^7.4.5",
-    "@babel/runtime": "^7.4.5",
+    "@babel/core": "^7.6.0",
+    "@babel/runtime": "^7.6.0",
+    "@react-native-community/eslint-config": "^0.0.5",
     "babel-jest": "^24.8.0",
+    "eslint": "^6.1.0",
     "fs-extra": "^7.0.1",
     "jest": "^24.8.0",
```

Comment:
this and babel-jest at 24.8.0 are specifically problematic and are best at 24.9.0 now (I think yarn handles the range well but npm does not and you get stuck on the old version which has terrible performance) with node v12.4-12.8 among other problems

Reply:
Updated both to 24.9.0 - also I always use yarn!